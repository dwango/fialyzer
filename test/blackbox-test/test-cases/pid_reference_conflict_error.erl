-module(pid_reference_conflict_error).
-export([main/1]).

main(X) ->
    ref_to_list(X),
    is_process_alive(X).
