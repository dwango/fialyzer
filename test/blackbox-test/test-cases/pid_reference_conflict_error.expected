TODO:filename:6: Type error: type mismatch;
  found   : reference()
  required: pid()
there is no solution that satisfies subtype constraints
